public/index.pug

```diff
@@ -351,6 +337,16 @@ html(lang='en-US')
                   path(
                     d='M11.2 0a.8.8 0 0 0-.8.8v11.4L7.26 9.44a.803.803 0 0 0-1.13.074l-1.05 1.2a.8.8 0 0 0 .073 1.13l6.33 5.54a.795.795 0 0 0 1.05 0l6.32-5.54a.8.8 0 0 0 .074-1.13l-1.05-1.2a.804.804 0 0 0-1.13-.074l-3.14 2.76V.8a.8.8 0 0 0-.8-.8zm-8 20.8a.8.8 0 0 0-.8.8v1.6a.8.8 0 0 0 .8.8h17.6a.8.8 0 0 0 .8-.8v-1.6a.8.8 0 0 0-.8-.8z'
                   )
+              a.report__icon.grid-item__button(
+                href=`https://github.com/simple-icons/simple-icons/issues/new?labels=icon+outdated&template=icon_update.md&title=Update%20icon%20for%20${icon.title}`,
+                role='button'
```

Comment:
Is missing a title here?

```suggestion
                role='button'
                title=`Report that ${icon.title} icon is outdated`
```

Reply:
I didn't miss it, I just didn't add it.
If we were to add it, I would prefer 'Report ${icon.title} as outdated'

Can you change the suggestion so we can commit it?

Reply:
Done 🔧